easybuild/easyconfigs/c/cairomm/cairomm-1.14.4-GCC-11.3.0-cairo-1.17.4.eb

```diff
@@ -0,0 +1,39 @@
+# Updated to MesonNinja as the autogen.sh complained.
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MesonNinja'
+
+name = 'cairomm'
+version = '1.14.4'
+versionsuffix = '-cairo-1.17.4'
```

Comment:
Why introduce a suffix? We only have one cairo for this toolchain (and i see no reason to introduce more, that's would just be painful).

Reply:
The idea was the following: `cairomm` is the interface and it can be used for different versions of `cairo`. So unless there is really only one version of `cairo` per toolchain and we are not changing that, we can remove it. 
It was one of these times where I was not quite sure about which program versions are *fixed* within a toolchain, and which one can have several versions. Given the information you provided, I have removed it. Thanks for your help.

Reply:
> only one version of

this is what we always strive for, for all dependencies. Multiple variants of dependencies are always a pain to deal with going forward, as it creates a fork of incompatible software within a toolchain for all future. Only sometimes unfortunately an unavoidable pain.

Reply:
I am happy to remove the 1.16 version but I really do need the 1.14 one (see below). The problem is their change of the API.

Reply:
1.14.4 removed, so no longer relevant